### PR TITLE
Issue #212 Create Issue Template To Provide Guidance on Writing Issues

### PR DESCRIPTION
**Pull request by vuinguyen**
Implements issue #212 

This PR creates an issue template to help folks write better issues.

Can I get a review @clc80 and @ceceliacreates when you get a chance please?

### CODE REVIEW

File: issue_template.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Recommendations for writing a good issue:
+
+## TODO: Give this issue a meaningful title
+* [iOS | Android] - [Short Description]
+* Example: iOS - UI - Add Submit Button to Detail Screen
+
+## TODO: Add more details to describe what needs to be done. Include wireframes if applicable.
+
+## TODO: If a PR already exists, link it to this issue
```
Comment (clc80) [inline issue_template.md:9]:
```suggestion
## If a PR already exists, link it to this issue
```

Do we still need the words "TODO" on this. Seems to be already addressed in list

File: issue_template.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+* Do this under the Development setting gear (on the right)
```
Comment (clc80) [inline issue_template.md:10]: maybe a screenshot of the settings gear would be better. I had no idea what the development setting gear is.

**Review by clc80 — CHANGES_REQUESTED** [review]
Some comments to consider.

File: issue_template.md (hunk 2 above)
Comment (vuinguyen) [reply issue_template.md]: I added a link that shows how to do this instead. The link shows how to get to the development setting gear. I have no idea how to add a screenshot in markdown.

File: issue_template.md (hunk 1 above)
Comment (vuinguyen) [reply issue_template.md]: I removed TODO: from the entire file

**Comment by vuinguyen** [comment]
@clc80 I made the requested changes. Please review again when you get a chance

File: issue_template.md (hunk 2 above)
Comment (vuinguyen) [reply issue_template.md]: No wait, I think I do remember how to add a screenshot in markdown. Think just need to drag it into the file. But anyway, I still think the link to the documentation is better

**Comment by ceceliacreates** [comment]
HI @clc80 I reviewed and approved, can you give one final approval so we can merge? Thank you!

**Review by clc80 — APPROVED** [review]

**Pull request merged**